### ISSUE TITLE
bug(mat-chip-list): Screen reader does not announce selection change

### ISSUE THREAD

**Issue by hatta2** [angular/components#21134]
#### Reproduction

Use StackBlitz to reproduce your issue: 
https://stackblitz.com/angular/koopndygyyjy?file=src%2Fapp%2Fchips-autocomplete-example.ts

Steps to reproduce:
1. Visit https://material.angular.io/components/chips/examples or StackBlitz
2. Enable ChromeVox for Chrome OS 86.0.4240.198
3. Keyboard navigate to a chip
4. Press space bar a few times to toggle selection state on or off. 
 
#### Expected Behavior

Selection state change is announced.

#### Actual Behavior

Selection state change is not announced.

#### Environment

- Angular: 11.0
- CDK/Material: Material
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Chrome OS 86.0.4240.198

**Comment by jelbourn** [comment]
I think this might be a bug with ChromeVox itself. I tested [MatSelectionList](https://material.angular.io/components/list/overview#selection-lists), the [listbox ARIA blueprints](https://static.corp.google.com/ariablueprints/), and [a W3C example](https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html#ex2_label), and none of them read the selected state change upon interaction. Do you have any working listbox interactions on ChromeVox to compare against?

**Comment by crisbeto** [comment]
Closing due to inactivity.
